src/Dfc.CourseDirectory.Functions/FixVenues/CorruptionType.cs

```diff
@@ -0,0 +1,13 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Dfc.CourseDirectory.Functions.FixVenues
+{
+    [JsonConverter(typeof(StringEnumConverter))] // https://stackoverflow.com/questions/45513772/serialize-enum-as-string-in-json-returned-from-azure-function/47564570#47564570
```

Comment:
Does this really require a comment?

Reply:
If I read it without the comment I wouldn't make the jump that this is specifically to control the http/json of the azure function.
Perhaps I need to make that clearer...

Going with this comment instead:
```
[JsonConverter(typeof(StringEnumConverter))] // Use strings in azure function json response. https://stackoverflow.com/a/47564570
```